common/utils.js

```diff
@@ -453,6 +453,17 @@ const getItsEdgeContract = (chainConfig) => {
     return itsEdgeContract;
 };
 
+const getItsEdgeChains = (config, excludeChainName) => {
+    return Object.keys(config.chains).filter((chain) => getItsEdgeContract(config.chains[chain]) && chain !== excludeChainName);
```

Comment:
The `axelarId` field should be used for the chain name since it's the actual value on-chain, not the config chain name key. Use `Object.values()` and map to the `axelarId` at the end

To avoid scenarios where config key differs from axelarId, you can add validation on the config checks. Still, we should always explicitly use the `axelarId`